styles.css

```diff
@@ -100,8 +103,8 @@ a {
     padding: 8px 12px;
     border-radius: 5px;
     margin: 0 12px 1.5rem  0;
-    border: 0.1rem solid;
-    box-shadow: inset 0.2rem 0.2rem rgba(71, 11, 11, 0.6);
+    border: 0.1rem solid rgba(71, 11, 11, 0.6);
```

Comment:
hmmmh 😄 it´s not wrong but it´s not necessary to use rem for a small boarder...i would recommend just 1px 
but that´s your choice 😄

Reply:
you are right, haha - it ended now with 2px each